internal/pkg/cli/svc_init_test.go

```diff
@@ -17,6 +19,13 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+var (
+	testInterval    = 10 * time.Second
+	testRetries     = 2
+	testTimeout     = 5 * time.Second
+	testStartPeriod = 0 * time.Second
+)
+
```

Comment:
I think those vars should be defined in `TestAppInitOpts_Execute` since they are only used there. Otherwise they are accessible in the whole `cli` package